src/app/pages/common/ix-tables/components/ix-table-expandable-row/ix-table-expandable-row.component.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  trigger, state, animate, style, transition,
+} from '@angular/animations';
+import {
+  Component, ChangeDetectionStrategy, Input,
+} from '@angular/core';
+import { Option } from 'app/interfaces/option.interface';
+
+@Component({
+  selector: 'ix-table-expandable-row',
+  templateUrl: './ix-table-expandable-row.component.html',
+  styleUrls: ['./ix-table-expandable-row.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  animations: [
+    trigger('rowExpand', [
+      state('collapsed', style({ height: '0px', minHeight: '0' })),
+      state('expanded', style({ height: '*' })),
+      transition('expanded <=> collapsed', animate('225ms cubic-bezier(0.4, 0.0, 0.2, 1)')),
```

Comment:
Animations on my system seem to feel rather sluggish. Maybe try a shorter animation time frame. 50ms yielded better results

Reply:
Did you click on the row itself or on expand button? I see a difference between them in the smoothness. Good catch, btw!